wiki/Guides/osu!mania_Modding_Guide/en.md

```diff
@@ -0,0 +1,177 @@
+# osu!mania Modding Guide
+
+This guide/tutorial will go through how to mod a beatmap in osu!mania. Modding is a relatively simple process and with enough practice, can lead to becoming a Beatmap Nominator.
```

Comment:
[..]with enough practice, can lead to becoming a Beatmap Nominator.
Is this really correct? I have the feeling it should be "[..] it can lead to becoming a Beatmap Nominator